proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -509,3 +509,21 @@ message GenerateLastHistoryReplicationTasksRequest {
 
 message GenerateLastHistoryReplicationTasksResponse {
 }
+
+message GetReplicationStatusRequest {
+}
+
+message GetReplicationStatusResponse {
+    repeated ShardReplicationStatus shards = 1;
+}
+
+message ShardReplicationStatus {
+    int32 shard_id = 1;
+    int64 max_replication_task_id = 2;
```

Comment:
Is this the max replication task id generated by the current cluster?

Reply:
Yes, that is correct.